spec/repositories/submission_repository_spec.rb

```diff
@@ -109,4 +109,36 @@
       end
     end
   end
+
+  describe '#to_invite' do
```

Comment:
My aim here was to ensure that this method returns correct submissions in every possible scenario so I don't need to test edge cases every time another class uses this method. Is there a better way to achieve that? Is there a better approach to testing things like this? 
Maybe bad code design is the source of this crammed test?